db/schema.rb

```diff
@@ -10,12 +10,19 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_09_18_142241) do
+ActiveRecord::Schema.define(version: 2019_09_23_145104) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
   enable_extension "plpgsql"
 
+  create_table "checks", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.integer "status"
+    t.string "checked_by"
+    t.datetime "created_at", null: false
```

Comment:
I don't know if creation date is a reliable "approval date". Can we be explicit about it? What happens if we reject a claim then decide to amend it and approve it?

Reply:
To be clear, I think this should happen now so we don't need to do data migrations later.

Reply:
I am beginning to wonder if we need multiple checks, but that comes with the added complication of which check do we treat as canonical? If something is approved, I don't think we should be able to unapprove it. It might be OK to approve something if it's previously been declined, or approve it after it's been escalated, but I think approved claims need to stay approved.

Reply:
Yeah, it's tricky I would argue that we should allow a claim to go from approved to rejected (ie. some new information has come to light) until the point that a payment has been requested? Maybe we'll need a state machine at that point?

Reply:
Oh ok. If checks are read-only, except on creation, can we enforce that somehow?

Reply:
Also, does it imply that `status` is actually like a `result`? `status` implies to me that it's mutable.

Reply:
I think I am with @erbridge on `status` I didn't think we would have multiple checks, but it makes way more sense, so `status` starts to feel wrong.